File: versions/V5/release_notes/v5.4.2/README_nawips5.4.2.c
			N-AWIPS 5.4.2.c Patch Release Notes
				4/23/97


This file describes the patch and provides instructions for its installation.

This patch contains some bug fixes and code improvements to version 5.4.2.b
These include:

	o  Some new capabilities for product generation including the
	   ability to move objects and draw lines in NMAP.  Also VFG
	   files can be named, saved and restored in NMAP.  The GUI is 
	   also improved.

	o  Some new capabilities have been added to NMAP.  Printing is now
	   available in NMAP. NOTE: Do NOT print model data overlaid on
	   image data.  This does not work properly.  The data access 
	   for observational data has been improved.  The station model 
	   for observational data is now table driven and uses GEMPAK 
	   aliases.  Map and lat/lon as selected from the GUI are now 
	   properly applied to the model data.

	o  A new decoder for Supplemental Climatological Data.  Also,
	   the flash flood guidance decoder has been revised.

	o  A new decoder, dcmsfc, that decodes ship, and buoy reports
	   is now available.

	o  Revised ths surface stations table to include many more foreign
	   METAR reporting stations.

	o  Removed changes to turbulence symbols plotting that were made
	   in 5.4.2.b.

See the N-AWIPS logs for additional details.  This patch begins on 04/01/97.

This patch requires some recompiling and linking of all GEMPAK programs,
GUI programs, and  device drivers.  A script has been written
to perform all of the necessary compiles and links.  It is called:

	patch_build_5.4.2.c 

The file is located in $GEMPAK/build.

To execute the script and save its output in a file type:

	cd $GEMPAK/build
	patch_build_5.4.2.c >&! PATCH_$OS & ; tail -f PATCH_$OS

The output of the script will be written to PATCH_$OS.

File: versions/V5/release_notes/v5.4.2/README_nawips5.4.2.f
			N-AWIPS 5.4.2.f Patch Release Notes
				6/4/97


This file describes the patch and provides instructions for its installation.

This patch contains some bug fixes and code improvements to version 5.4.2.e
These include:

	o  Several corrections have been made to the product generation
	   software.  These include corrections to allow proper editing
	   of elements created using the VG driver.  Contour line editing
	   has also been corrected in NMAP.

	o  Multiple line text creation has been added to GPLT and NMAP.

	o  Several new MPC products have been added to NWX.

	o  New AFOS alarms requested by the HPC have been added.

	o  NMAP model product attributes have been improved.


See the N-AWIPS logs for additional details.  This patch includes
development from 05/22/97 through 6/3/97.

This patch requires some recompiling and linking of GEMPAK programs,
GUI programs, and  device drivers.  A script has been written
to perform all of the necessary compiles and links.  It is called:

	patch_build_5.4.2.f 

The file is located in $GEMPAK/build.

To execute the script and save its output in a file type:

	cd $GEMPAK/build
	patch_build_5.4.2.f >&! PATCH_$OS & ; tail -f PATCH_$OS

The output of the script will be written to PATCH_$OS.


